test/ibmq/test_basic_server_paths.py

```diff
@@ -36,6 +42,17 @@ def setUpClass(cls, providers):
         super().setUpClass()
         cls.providers = providers  # Dict[str, AccountProvider]
         cls.last_week = datetime.now() - timedelta(days=7)
+        # Setup list of jobs to check have been deleted.
+        cls.jobs_to_delete: List[Tuple[AccountProvider, str]] = []
+
+    def tearDown(self) -> None:
+        # Ensure all jobs are deleted.
+        for provider, job_id in self.jobs_to_delete:
+            try:
+                provider.runtime.delete_job(job_id)
+            except RuntimeJobNotFound:
+                pass
```

Comment:
Could you add a log statement here like "Unable to delete runtime job: <job_id>" instead of pass?

Reply:
[Updated! ](https://github.com/Qiskit/qiskit-ibmq-provider/pull/1012/files#diff-762f01fbe80fb731096879850a9910fe15c4286292d868375a0b9fdcbc576c4eR56) The language is slightly different, I can modify in next push.